docs/known-issues.md

```diff
@@ -120,7 +120,11 @@ Enabling Mandatory ASLR affects the MSYS2 core library, which is relied upon by
 
 Electron enables hardware accelerated graphics by default, but some graphics cards have issues with hardware acceleration which means the application will launch successfully but it will be a black screen.
 
-**Workaround:** if you set the `GITHUB_DESKTOP_DISABLE_HARDWARE_ACCELERATION` environment variable to any value and launch Desktop again it will disable hardware acceleration on launch, so the application is usable.
+**Workaround:** if you set the `GITHUB_DESKTOP_DISABLE_HARDWARE_ACCELERATION` environment variable to any value and launch Desktop again it will disable hardware acceleration on launch, so the application is usable. Here are the steps to set the environment variable in PowerShell:
+
+1. Open PowerShell
+2. Run the command `$env:GITHUB_DESKTOP_DISABLE_HARDWARE_ACCELERATION=1`
```

Comment:
Is a space needed after the `$` sign?

Reply:
@steveward could you confirm this is correct as-is or needs to be changed?
```suggestion
2. Run the command `$ env:GITHUB_DESKTOP_DISABLE_HARDWARE_ACCELERATION=1`
```

Reply:
A space isn't needed -- [docs for reference](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_environment_variables?view=powershell-7):

`$Env:<variable-name> = "<new-value>"`